userguide/advanceduse/reverse-tethering.rst

```diff
@@ -1,34 +1,34 @@
 Reverse tethering
 =================
 
-Some users may not have an available wifi connection for their phone nor a data subscription from their mobile provider.
-This short tutorial will help you to connect your Ubuntu Touch to your computer to access internet.
+This tutorial helps you to connect your UBports device to your computer to access the Internet on it.
+This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your UBports device.
 
-Prerequisite: Phone is connected to the computer with usb and developer mode enabled.
+Prerequisite: A UBports device with "USB debugging" and "Developer mode" on, attached via USB to the computer connected to the Internet.
```

Comment:
Is there such a thing as USB debugging in UT?

Reply:
I don't really know. Don't have my device here to check.
Is it "USB and developer mode"? I think I checked something else in the documentation to see, but that is neither here nor there.

Reply:
@doniks You are correct, reverting in all PRs.

Reply:
thx for the UBports/UT edit

leaves us with the "USB debugging" ... I'm pretty sure this is what it's called in Android, and I'm pretty sure it's just called "Developer mode" in UT

Reply:
Ah yes, I was reading it all as one clause. The "USB" is how it connects, not any of/the mode.